source_gen/pubspec.yaml

```diff
@@ -1,18 +1,18 @@
 name: source_gen
-version: 0.9.10+4
+version: 1.0.0
 description: >-
   Source code generation builders and utilities for the Dart build system
 repository: https://github.com/dart-lang/source_gen
 
 environment:
-  sdk: ">=2.11.999 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  analyzer: '>=0.42.0 <2.0.0'
+  analyzer: ^1.0.0
```

Comment:
Do you know if you are relying on any of the changes to make some types be non-nullable now? (do the tests pass if you pin to this exactly?)

Reply:
will do a sanity run locally @ v1.0.0

Reply:
dart_style requires ^1.10 so I bumped to that.

Tested locally. All is good!